### PR TITLE
Hipehdf5

### PR DESCRIPTION
**Pull request by devine73**
As announced in the ASWG meeting on Thursday 9th, the HDF5 reader for the HDF5 data format from the LAPP for R1 (direct conversion from simtel file).
To contact the author:
      pierre.aubert@lapp.in2p3.fr

### CODE REVIEW

File: ctapipe/io/simteleventsource.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def __init__(self, config=None, parent=None, **kwargs):
             self.input_url,
             allowed_telescopes=self.allowed_tels if self.allowed_tels else None,
             skip_calibration=self.skip_calibration_events,
-            zcat=not self.back_seekable,
+            #zcat=not self.back_seekable,
```
Comment (vuillaut) [inline ctapipe/io/simteleventsource.py]: why commenting this line?

Comment (MaxNoe) [reply ctapipe/io/simteleventsource.py]: yes, this should not be changed. First: it's completely unrelated and second: why?

Comment (devine73) [reply ctapipe/io/simteleventsource.py]: I commented this line to ensure the tests passed on my computer.

Comment (MaxNoe) [reply ctapipe/io/simteleventsource.py]: You dont Change unrelated code to make tests pass. This is what tests are for!

**Comment by kosack** [comment]
Event sources other than SimTelEventSource are now in separate repos to avoid complicating ctapipe with non-standard formats, could we put this in it's own repo as a plugin (similar to ctapipe_io_lst, ctapipe_io_magic, ctapipe_io_nectarcar, etc)?

**Comment by kosack** [comment]
Also, it may be better to wait until we define a common DL1 format - we don't want to have more than one of them in the end! If this format has nice features, I would suggest please add comments to the issue #1059 or a new issue to explain what should be added to the common format.

**Comment by MaxNoe** [comment]
Isn't this R0/DL0?

**Comment by kosack** [comment]
> Isn't this R0/DL0?

Monte-Carlo doesn't really fit yet with the final data levels - I guess it's more R0 than R1/DL0 though.

File: ctapipe/io/simteleventsource.py (hunk 1 above)
Comment (kosack) [reply ctapipe/io/simteleventsource.py]: the tests for that work fine on my machine (and on travis), so it shouldn't be removed

Comment (MaxNoe) [reply ctapipe/io/simteleventsource.py]: This hints to an outdated eventio version. Use `python setup.py test` or do `pip install -e .` again, then it should upgrade eventio.

**Comment by devine73** [comment]
Ok, I will make a plugin. Do you want something like ctapipe_io_R0 or maybe ctapipe_io_MCR0 to express this is a Monte-Carlo data format ? Or other names...

**Comment by kosack** [comment]
This code is moving to a plugin, so I'll close this PR for now

**Pull request closed without merging**